theme/basics/template-hierarchy.md

```diff
@@ -424,79 +189,35 @@ Rendering an attachment page (`attachment` post-type) uses the following path:
 5.  `single.php`
 6.  `singular.php`
 7.  `index.php`
--->
-1. `{MIME-type}.php` – 任意の[MIME タイプ（メディアタイプ）](https://ja.wikipedia.org/wiki/%E3%83%A1%E3%83%87%E3%82%A3%E3%82%A2%E3%82%BF%E3%82%A4%E3%83%97) (例: `image.php`, `video.php`, `pdf.php`)。`text/plain` ならば以下の順で表示されます。
-    1.  `text-plain.php`
-    2.  `plain.php`
-    3.  `text.php`
-2. `attachment.php`
-3. `single-attachment-{slug}.php` – 例えば、添付ファイルのスラッグが `holiday` であれば、WordPress は `single-attachment-holiday.php` を探します。
-4. `single-attachment.php`
-5. `single.php`
-6. `singular.php`
-7. `index.php`
-
-<!-- 
+
 ### Embeds
--->
-### 埋め込み
 
-<!-- 
 The embed template file is used to render a post which is being embedded. Since 4.5, WordPress uses the following path:
```

Comment:
<!-- 
The embed template file is used to render a post which is being embedded. Since 4.5, WordPress uses the following path:
-->
埋め込みテンプレートファイルは、埋め込み対象の記事を表示するために使用されます。4.5 以降、WordPress は以下の順で表示します。